exercises/pangram/description.md

```diff
@@ -2,5 +2,6 @@ Determine if a sentence is a pangram. A pangram (Greek: παν γράμμα, pan
 "every letter") is a sentence using every letter of the alphabet at least once.
 The best known English pangram is "The quick brown fox jumps over the lazy dog."
 
-The alphabet used is ASCII, and case insensitive, from 'a' to 'z'
-inclusively.
+The alphabet used consists of ASCII letters `a` to `z`, inclusive, and is case
+insensitive. Per [#741](https://github.com/exercism/problem-specifications/pull/741),
```

Comment:
Sure thing, pull request updated. No tests need to be updated, since the change is to just clarify that tests should be ASCII-clean, but otherwise can contain characters that are not letters (which should be just skipped). That is, current tests already contain only ASCII symbols already.